src/components/collectibles/CollectibleTile.jsx

```diff
@@ -0,0 +1,64 @@
+import { h } from 'preact'
+import { useEffect, useState } from 'preact/hooks'
+import cn from 'classnames'
+
+import { getFrameFromGif } from './collectibleHelpers'
+import styles from './CollectibleTile.module.css'
+
+const preload = async (src) => (
```

Comment:
probably we should pull this out into a util https://github.com/AudiusProject/bedtime/blob/0cf13a19a1fa0a1647aeec97f87f77eab0cf8341/src/components/artwork/Artwork.jsx#L12